packages/distcc/Makefile.am.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/Makefile.in b/Makefile.in
+index 3ea595c..0fb1efd 100644
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -21,7 +21,7 @@ PYTHON_CFLAGS = @PYTHON_CFLAGS@
+ POPT_CFLAGS = @POPT_CFLAGS@
+ POPT_INCLUDES = @POPT_INCLUDES@
+ 
+-LDFLAGS = @LDFLAGS@
++LDFLAGS = @LDFLAGS@ -llog -pie
+ CC = @CC@
+ CPP = @CPP@
+ # We add a few cppflags.  -Isrc is so that config.h can be found in the build
+@@ -67,7 +67,7 @@ GNOME_LIBS = @GNOME_LIBS@
+ 
+ LIBS = @LIBS@
+ 
+-DESTDIR =
++DESTDIR = /data/data/com.termux/files
```

Comment:
Shouldn't it be /data/data/com.termux/files/usr?

If that is the case then it would also be better to use @TERMUX_PREFIX@, see for example this commit:
https://github.com/termux/termux-packages/commit/e0ab589ef9582a9d17ae90c5b1d4309cc0af4c67